sormas-api/src/main/java/de/symeda/sormas/api/sormastosormas/SormasToSormasShareableDto.java

```diff
@@ -15,18 +15,46 @@
 
 package de.symeda.sormas.api.sormastosormas;
 
+import de.symeda.sormas.api.ImportIgnore;
 import de.symeda.sormas.api.user.UserReferenceDto;
+import de.symeda.sormas.api.utils.Required;
 import de.symeda.sormas.api.utils.pseudonymization.PseudonymizableDto;
 
-public abstract class SormasToSormasShareableDto extends PseudonymizableDto {
-
-	public abstract SormasToSormasOriginInfoDto getSormasToSormasOriginInfo();
-
-	public abstract void setSormasToSormasOriginInfo(SormasToSormasOriginInfoDto originInfo);
+import javax.validation.Valid;
 
-	public abstract boolean isOwnershipHandedOver();
-
-	public abstract UserReferenceDto getReportingUser();
+public abstract class SormasToSormasShareableDto extends PseudonymizableDto {
 
-	public abstract void setReportingUser(UserReferenceDto user);
+	public static final String REPORTING_USER = "reportingUser";
+	public static final String SORMAS_TO_SORMAS_ORIGIN_INFO = "sormasToSormasOriginInfo";
+	public static final String OWNERSHIP_HANDED_OVER = "ownershipHandedOver";
+	@Valid
+	protected SormasToSormasOriginInfoDto sormasToSormasOriginInfo;
+	protected boolean ownershipHandedOver;
+	@Required
+	protected UserReferenceDto reportingUser;
```

Comment:
I wouldn't move reporting user field into SormasToSormasShareableDto because it's not related to share. 
It's enough to have the `setReportingUser` abstract method because only that one is used in S2S.

Reply:
Yes good point, I was thinking about the same thing. Will add it later thanks!

Reply:
fixed